apps/vs-code-designer/src/app/commands/funcCoreTools/uninstallFuncCoreTools.ts

```diff
@@ -2,10 +2,52 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
-import { extensionCommand } from '../../../constants';
-import type { PackageManager } from '../../../constants';
-import { commands } from 'vscode';
+import { PackageManager, wingetFuncPackageName, funcPackageName } from '../../../constants';
+import { ext } from '../../../extensionVariables';
+import { localize } from '../../../localize';
+import { executeCommand } from '../../utils/funcCoreTools/cpUtils';
+import { tryGetLocalFuncVersion } from '../../utils/funcCoreTools/funcVersion';
+import { tryGetInstalledFuncCoreToolsBrewPackageName } from '../../utils/packageManagers/getBrewPackageName';
+import { getGlobalSetting } from '../../utils/vsCodeConfig/settings';
+import type { IActionContext, IAzureQuickPickItem } from '@microsoft/vscode-azext-utils';
+import type { FuncVersion } from '@microsoft/vscode-extension';
 
-export async function uninstallFuncCoreTools(packageManagers?: PackageManager[]): Promise<void> {
-  await commands.executeCommand(extensionCommand.azureFunctionsUninstallFuncCoreTools, packageManagers);
+export async function uninstallFuncCoreTools(context: IActionContext, packageManagers?: PackageManager[]): Promise<void> {
+  context.telemetry.properties.uninstallFuncCoreTools = 'true';
+
+  let packageManager = packageManagers ? packageManagers[0] : null;
+  if (packageManager == undefined || packageManager == null) {
+    const platformPackageManagerKey = 'PlatformPackageManager';
+    packageManager = getGlobalSetting(platformPackageManagerKey);
+    context.telemetry.properties.packageManager = packageManager;
+  }
+
+  if (packageManagers && packageManagers.length === 0) {
+    throw new Error(
+      localize('notInstalled', 'Cannot uninstall Azure Functions Core Tools because it is not installed with supported package manager.')
+    );
+  } else if (packageManagers && packageManagers.length > 1 && context.telemetry.properties.multiFunc) {
+    const placeHolder: string = localize('multipleInstalls', 'Multiple installs of the func cli detected. Select the one to uninstall');
+    const picks: IAzureQuickPickItem<PackageManager>[] = packageManagers.map((pm) => {
+      return { label: localize('uninstall', 'Uninstall {0} package', pm), data: pm };
+    });
+    packageManager = (await context.ui.showQuickPick(picks, { placeHolder, stepName: 'multipleFuncInstalls' })).data;
+  }
+
+  switch (packageManager) {
+    case PackageManager.npm:
```

Comment:
Can we not call the azure functions command for the npm and brew? The same way you did for the install

Reply:
winget is not a supported package manager for func core tools extension, which is why I had to also copy the AzureQuickPickItem. I can't use the extension command because then AzureQuickPickItem prompt will show twice and be redundant.
Scenario: 
2 azure function core tools version installed (installed with winget and npm).
<img width="337" alt="image" src="https://github.com/Azure/LogicAppsUX/assets/12243528/7ff788d1-b63d-4c1e-b98c-709add132281">
<img width="453" alt="image" src="https://github.com/Azure/LogicAppsUX/assets/12243528/212d1c02-7020-4de7-8091-60c626727f24">